pages/studentAssessmentExam/studentAssessmentExam.js

```diff
@@ -37,22 +37,50 @@ router.get('/', function (req, res, next) {
           maxsize,
```

Comment:
Done with this commit: https://github.com/PrairieLearn/PrairieLearn/pull/6109/commits/5bb0f1977c1dc794132ff1abc6a0cd4dfc929508

We'll still have to change `studentAssessmentInstanceExam.js`. I'm a little unsure of how to do it correctly since the callback code there doesn't have async versions